tests/wpunit/Fee/PlanListingTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * Includes tests for the Fees class.
+ */
+
+namespace Fee;
+
+use WPBDP\Tests\WPUnitTestCase;
+use Codeception\Util\Debug;
+use WPBDP__Fee_Plan;
+
+/**
+ * Tests for fee Listing
+ */
+class PlanListingTest extends WPUnitTestCase {
+
+	/**
+	 * @var \WpunitTester
+	 */
+	protected $tester;
+
+	public function testFeePlanVisibility() {
```

Comment:
Let's stick with underscore function names instead of camelcase

Reply:
@stephywells on the unit test cases, it must be camel case